sphinxcontrib/prettyspecialmethods.py

```diff
@@ -207,8 +211,10 @@ def apply(self):
 
 
 def show_special_methods(app, what, name, obj, skip, options):
+    # type: (Sphinx, str, str, Any, bool, Dict[str, Any]) -> Optional[bool]
```

Comment:
Can you use the Python 3.5 syntax here?

Reply:
@eric-wieser Using annotations makes the signature too long for one line. Do you have any preference for splitting it up? I usually favor this format:

```
def show_special_methods(
    app: Sphinx,
    what: str,
    name: str,
    obj: Any,
    skip: bool,
    options: Dict[str, Any],
) -> Optional[bool]:
```

Reply:
Also, switching to the type annotations causes the code to fail at runtime if the `TYPE_CHECKING` is used to prevent the types from being imported. Which is fine with me, but it means importing things unconditionally.

Reply:
Ah, I forgot about that. It doesn't really matter then, do whichever you think is best

Reply:
@eric-wieser I opted to drop the `if TYPE_CHECKING`. Sphinx is still supporting down to 3.5, and it's not like the imports are particularly expensive at runtime.